samples/snippets/analyze_content_stream_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import conversation_management
+import conversation_profile_management
+import participant_management
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+CONVERSATION_PROFILE_DISPLAY_NAME = "fake_conversation_profile"
```

Comment:
This should probably use a UUID: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#testing-guidelines

Reply:
Sorry if I misunderstood, this is not a resource name, it's just a display name. Resource name is generated and returned by the rpc response.

Reply:
Fixed.